packages/server/test/cache_helper.js

```diff
@@ -0,0 +1,19 @@
+const supportedConfigFiles = [
```

Comment:
Should we use `CYPRESS_CONFIG_FILES` here?

Also pls make the new files TS - it will bring a lot of long term value, there's not reason not to.

Reply:
Done in bb00ece